queries.yaml

```diff
@@ -328,13 +328,13 @@ post_process:
   - fn: TileStache.Goodies.VecTiles.transform.drop_features_where
     params:
       source_layer: landuse
-      start_zoom: 13
+      start_zoom: 6
       where: >-
         properties.get('mz_is_building') is True
   - fn: TileStache.Goodies.VecTiles.transform.drop_properties
     params:
       source_layer: landuse
-      start_zoom: 13
+      start_zoom: 6
```

Comment:
> So anytime we modify the landuse start zoom elsewhere in the codebase we need to update it here? 

Yes. More specifically any time we update the starting zoom in the [landuse query](https://github.com/mapzen/vector-datasource/blob/9aa9507ab5489b8613127e0db6c87a5e2d1c8a65/queries/landuse.jinja2#L32) for when we include the mz_is_building property.

> Why was it set to 13 before?

We start including buildings at z13.